python/lance/tests/test_api.py

```diff
@@ -36,6 +36,16 @@ def test_simple_round_trips(tmp_path: Path):
     assert table == actual
 
 
+def test_head(tmp_path: Path):
```

Comment:
can we add a test to make sure duckdb works with this dataset

Reply:
there is an existing unit test that runs a duckdb query (how i discovered the problem originally)